runtimes/php-8.1/README.md

```diff
@@ -72,26 +72,8 @@ You can also make changes to the example code and apply the changes with the `do
 ```php
 <?php
 
-return function($req, $res) {
-    $res->send('Hello Open Runtimes 👋');
-};
-```
-
-- The `res` parameter has two methods:
-
-    - `send()`: Send a string response to the client.
-    - `json()`: Send a JSON response to the client.
-
-You can respond with `json()` by providing object:
```

Comment:
Why did we remove this ? Shouldn't we still keep the documentation for the `context` object ?

Reply:
It's consistent, so I removed it from runtime readme. I want to add it to main README.

Reply:
okay 👍